### PR TITLE
Renaming `interface{}` to `any` Part 1

### PR DESCRIPTION
**Pull request by Tang8330**
* Taking over this PR: https://github.com/artie-labs/transfer/pull/354
* `any` is an alias of `interface{}`

### CODE REVIEW

File: lib/cdc/util/geometry.go [hunk 1]
```diff
@@ -31,10 +31,10 @@ const Point GeometricShapes = "Point"
 // ParseGeometry takes in a map[string]interface{} and returns a GeoJSON string.
```
Comment (nathan-artie) [inline lib/cdc/util/geometry.go:31]: This comment can be updated.

File: lib/cdc/util/parse.go [hunk 2]
```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/artie-labs/transfer/lib/debezium"
 )
 
-// ParseField returns a `parsedValue` as type interface{}
-func parseField(field debezium.Field, value interface{}) interface{} {
+// ParseField returns a `parsedValue` as type any
+func parseField(field debezium.Field, value any) interface{} {
```
Comment (nathan-artie) [inline lib/cdc/util/parse.go:15]: Do you want to change the one at the end too?

File: lib/cdc/util/geometry.go (hunk 1 above)
Comment (Tang8330) [reply lib/cdc/util/geometry.go:31]: cool

File: lib/cdc/util/parse.go (hunk 2 above)
Comment (Tang8330) [reply lib/cdc/util/parse.go:15]: sure

**Review by nathan-artie — APPROVED** [review]

**Pull request merged**